MigrateServicePreferenceFromLegacy/handler.ts

```diff
@@ -12,6 +12,8 @@ import {
 } from "@pagopa/io-functions-commons/dist/src/utils/cosmosdb_model";
 import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
 import * as a from "fp-ts/lib/Array";
+import * as e from "fp-ts/lib/Either";
```

Comment:
Can we agree on some naming standard? Should we always use capital case letters to import fp-ts o lower case letters? In my opinion it is a bit ugly that sometimes we use `o` and sometimes `O` or other letters to refer to fp-ts.

Reply:
I agree on capital letters, it's how the `fp-ts` community seems to converge

Reply:
Yes sure, I missed something that was already defined but I''ll fix it.